tsconfig.json

```diff
@@ -45,7 +45,7 @@
     "noFallthroughCasesInSwitch": true /* Report errors for fallthrough cases in switch statement. */,
     "noUncheckedIndexedAccess": true /* Include 'undefined' in index signature results */,
     // "noPropertyAccessFromIndexSignature": true,  /* Require undeclared properties from index signatures to use element accesses. */
-    "importsNotUsedAsValues": "error",
+    // "importsNotUsedAsValues": "error",
```

Comment:
Is this rule needed or not?

Reply:
Yes, was going to enquire, what's the reason for this change?

Reply:
It's actually just marked as deprecated, there was no need to change it so I changed it back. The editor is showing in error state. It will become an issue when we upgrade to ts 5.5.